CHANGELOG.tpl

```diff
@@ -7,6 +7,18 @@ Platform support legends:
     * Note: the papercuts will majorly be with the build process. Things might be very rough to integrate as no polish at all has been given yet.
 * ❌ = tier 3 support. It doesn't work just yet, but we plan to make it work.
 
+## [0.5.1] - 2022-21-09
+
+<details>
+    <summary>git-conventional changelog</summary>
+{{git-cliff tag="v0.5.1" unreleased=true}}
```

Comment:
On the 0.5.0 tag you can remove the `unreleased=true` this should fix the changelog